app/models/offspring.rb

```diff
@@ -0,0 +1,8 @@
+class Offspring < ApplicationRecord
+  belongs_to :user
+
+  class GradedOffspring < Offspring
```

Comment:
I like the Sergio variant to, separate in class. However, it is debatable how confusing it can be. In the same ManageIQ there are many classes within classes and this depends on the complexity of the classes in a model. I think we should be more flexible in the comments and revisions, variants of each must be respected, this is programming, not the Constitution, not everything is written ;-)